meta-grpc-client/src/main/java/net/opentsdb/aura/metrics/meta/grpc/protobuf/TimeseriesLite.java

```diff
@@ -0,0 +1,65 @@
+package net.opentsdb.aura.metrics.meta.grpc.protobuf;
+
+import com.google.protobuf.ByteString;
+import com.google.protobuf.CodedOutputStream;
+import com.google.protobuf.MessageLite;
+import com.google.protobuf.Parser;
+
+import java.io.OutputStream;
+
+public class TimeseriesLite implements MessageLite {
```

Comment:
This is exactly same as `BitmapGroupResultLite `. We can pull it up to a base class and reuse it.

Reply:
ohh... `BitmapGroupResultLite ` is extending another base class. In that case we can create another Interface extending `MessageLite `  and add these as `default` methods and use that. The below code snippet would make it clear.

```
public interface DefaultMessageLite extends MessageLite {

  @Override
  default void writeTo(CodedOutputStream output){
    throw new UnsupportedOperationException();
  }
  ...
}

public class TimeseriesLite implements DefaultMessageLite {
  ...
}

public class BitmapGroupResultLite extends DefaultMetaTimeSeriesQueryResult.DefaultGroupResult implements DefaultMessageLite {
  ...
}
  
```

Reply:
Yeah was thinking of doing that.

Reply:
Food for thought.

Do we even need this empty classes like`DictionaryLite`, `TimeseriesLite` etc. This is definitely a code smell but we can live with it for now.